src/types/SortType.ts

```diff
@@ -0,0 +1,5 @@
+export enum SortType {
```

Comment:
it's actually not a _sort_ type, but "Status" or "FilterType"

Reply:
renamed